tree/ntuple/v7/inc/ROOT/RNTuple.hxx

```diff
@@ -378,6 +378,68 @@ private:
    /// Estimator of uncompressed cluster size, taking into account the estimated compression ratio
    std::size_t fUnzippedClusterSizeEst;
 
+public:
+   RNTupleFillContext(std::unique_ptr<RNTupleModel> model, std::unique_ptr<Detail::RPageSink> sink);
```

Comment:
Do we want this as a `public` constructor? Initially my prototype had a `private` constructor and `friend RNTupleWriter` (plus eventually `friend RNTupleParallelWriter`) so that only they are able to create fill contexts...

Reply:
I think I would prefer the private constructor.

Reply:
Okay, changed. As `RNTupleWriter` is already a `friend` for other reasons, no further updates needed.